### ISSUE TITLE
Automate Prebuilt Addons

### ISSUE THREAD

**Issue by shama** [shama/navelgazer#12]
Possibly using otto to prebuild binaries for supported versions of v8 on typically hard to compile environments (Windows and less so OSX).

**Comment by joshperry** [comment]
Yes, this is something I've struggled with in the node-serialport project. They use node-pre-gyp which basically sits on top of gyp in order to add pre-build binary delivery. It's kind of a pain to manage.

It should be pretty easy to add this functionality to the build.js script. Create an HTTPS URL using a base path and the calculated platform file name, check that URL for a published native binary, and pull it down into the proper directory. If a file doesn't exist at the well-known URL, then it can fall back to attempting the build from source like usual.
